### ISSUE TITLE
react-summernote doesn't support react 18

### ISSUE THREAD

**Issue by guptamanish07** [summernote/react-summernote#126]
![image](https://github.com/summernote/react-summernote/assets/51395568/dcb40318-137b-431a-ba7e-7a6d485a06c8)

**Comment by le-el** [comment]
Same error. How to solve this problem?

**Comment by ShivamPrajapati45** [comment]
yes this true i have also facing react 18 issue when installing summernote packeges. I recommed not to use summernote now bcoz it is officially said that it does not support react 18.
